__tests__/unit/lib/utils/repoGitDiff.test.js

```diff
@@ -169,60 +217,84 @@ describe(`test if repoGitDiff`, () => {
 
   test('can filter moved files', async () => {
     const output = [
-      'D      force-app/main/default/classes/Account.cls',
-      'A      force-app/account/domain/classes/Account.cls',
+      'force-app/main/default/classes/Account.cls',
+      'force-app/account/domain/classes/Account.cls',
     ]
-    child_process.__setOutput([output])
+    child_process.__setOutput([
+      [],
+      [`1${TAB}1${TAB}${output[0]}`],
+      [`1${TAB}1${TAB}${output[1]}`],
+    ])
     const repoGitDiff = new RepoGitDiff(
       { output: '', repo: '' },
       globalMetadata
     )
     const work = await repoGitDiff.getLines()
-    const expected = [output[1]]
+    const expected = [`${ADDITION}${TAB}${output[1]}`]
     expect(work).toStrictEqual(expected)
   })
 
   test('can filter case changed files', async () => {
     const output = [
-      'D      force-app/main/default/objects/Account/fields/TEST__c.field-meta.xml',
-      'A      force-app/main/default/objects/Account/fields/Test__c.field-meta.xml',
+      'force-app/main/default/objects/Account/fields/TEST__c.field-meta.xml',
+      'force-app/main/default/objects/Account/fields/Test__c.field-meta.xml',
     ]
-    child_process.__setOutput([output])
+    child_process.__setOutput([
+      [],
+      [`1${TAB}1${TAB}${output[0]}`],
+      [`1${TAB}1${TAB}${output[1]}`],
+    ])
     const repoGitDiff = new RepoGitDiff(
       { output: '', repo: '' },
       globalMetadata
     )
     const work = await repoGitDiff.getLines()
-    const expected = [output[1]]
+    const expected = [`${ADDITION}${TAB}${output[1]}`]
     expect(work).toStrictEqual(expected)
   })
 
   test('cannot filter renamed files', async () => {
     const output = [
-      'D      force-app/main/default/classes/Account.cls',
-      'A      force-app/main/default/classes/RenamedAccount.cls',
+      'force-app/main/default/classes/Account.cls',
+      'force-app/main/default/classes/RenamedAccount.cls',
     ]
-    child_process.__setOutput([output])
+    child_process.__setOutput([
+      [],
+      [`1${TAB}1${TAB}${output[0]}`],
+      [`1${TAB}1${TAB}${output[1]}`],
+    ])
     const repoGitDiff = new RepoGitDiff(
       { output: '', repo: '' },
       globalMetadata
     )
     const work = await repoGitDiff.getLines()
-    expect(work).toStrictEqual(output)
+    const expected = [
+      `${ADDITION}${TAB}${output[1]}`,
+      `${DELETION}${TAB}${output[0]}`,
+    ]
+    expect(work).toStrictEqual(expected)
   })
 
   test('cannot filter same name file with different metadata', async () => {
     const output = [
-      'D      force-app/main/default/objects/Account/fields/CustomField__c.field-meta.xml',
-      'A      force-app/main/default/objects/Opportunity/fields/CustomField__c.field-meta.xml',
+      'force-app/main/default/classes/Account.cls',
+      'force-app/main/default/classes/RenamedAccount.cls',
```

Comment:
This test is now testing something completely different (class renaming, instead of testing for the situation where the same field name is on 2 different object). I'm not sure to fully understand why?

Reply:
True, nice catch, I fix it